content/post/why-i-joined-sourced-francesc.md

```diff
@@ -0,0 +1,117 @@
+---
+author: campoy
+date: 2018-01-15
+title: "Why did I join source{d}? - Francesc Campoy"
+draft: true
+description: "The first post of a series on why multiple employees joined source{d}. This one is by Francesc Campoy."
+categories: ["why-i-joined"] 
```

Comment:
Should this post appear in our landing, under the "culture" category?
If so, it should be also categorized as `culture`

Reply:
done